### DISCUSSION TITLE
PE 13-plus, ricedroidoss and evolution x sync failing

### DISCUSSION THREAD

**Discussion by anandhan07** [ROM-builders/temporary#17075] [General]
Sync is failing for all people in the past 3 days for ROMs Pixel experience thirteen plus, ricedroidoss thirteen, evolution x tiramisu.

https://cirrus-ci.com/task/5452285627596800
https://cirrus-ci.com/build/6741433349046272
https://cirrus-ci.com/task/6146159337537536

https://cirrus-ci.com/task/6013696204341248
https://cirrus-ci.com/task/5860421035360256
https://cirrus-ci.com/task/5307902886412288

https://cirrus-ci.com/task/5895278721499136
https://cirrus-ci.com/task/6337557978415104

**Comment by alecchangod** [comment]
Also aospa for me
https://cirrus-ci.com/build/6322497944027136

**Comment by SwapnilVicky** [comment]
Also sync failing for blissrom typhoon  https://cirrus-ci.com/build/6252512089473024

**Comment by anandhan07** [comment]
@Apon77

**Comment by alternoegraha** [comment]
yeah, still sync fails for me too (as of today). here's a error:
```
error: vendor/qcom/opensource: Cannot remove project: uncommitted changes are present.
```
